src/custom-elements.js

```diff
@@ -0,0 +1,6 @@
+import register from 'preact-custom-element';
```

Comment:
Is it possible to import stylekit without any custom-elements code? If not it would be good to output two separate builds to save on bundle size

Reply:
I agree, this should be the way to go!

Reply:
- `stylekit.js`: contains the `SKAlert` class
- `stylekit.css`: contains all the compiled css
- `web-components.js` the compiled preact components, ready to use as Web Components.

Reply:
What do you think of also adding 

- `components.js`: contains the compiled preact components _without_ preact itself or the web components library

This is really what I'd need for web myself 😅